_includes/faq/auto-generate-unique-ids_v1.1.html

```diff
@@ -0,0 +1,21 @@
+To auto-generate unique row IDs, use the [`SERIAL`](serial.html) data type, which is an alias for [`INT`](int.html) with the `unique_rowid()` [function](functions-and-operators.html#id-generation-functions) as the [default value](default-value.html):
+
+~~~ sql
+> CREATE TABLE test (id SERIAL PRIMARY KEY, name STRING);
+~~~
+
+On insert, the `unique_rowid()` function generates a default value from the timestamp and ID of the node executing the insert, a combination that is likely to be globally unique except in extreme cases where a very large number of IDs (100,000+) are generated per node per second. In such cases, you should use a [`BYTES`](bytes.html) column with the `uuid_v4()` function as the default value instead:
+
+~~~ sql
+> CREATE TABLE test (id BYTES PRIMARY KEY DEFAULT uuid_v4(), name STRING);
+~~~
+
+Because `BYTES` values are 128-bit, much larger than `INT` values at 64-bit, there is virtually no chance of generating non-unique values.
+
+Alternatively, you can cast UUID as bytes and use the uuid_v4() function. 
```

Comment:
I'd move this above the "Because BYTES values..." sentence and rephrase a bit:

> Alternatively, you can use a `UUID` column with `uuid_v4()` function cast to UUID as the default value:

Then you can follow the example with:

> Because `BYTES` and `UUID` values are 128-bit, much larger than `INT` values at 64-bit, there is virtually no chance of generating non-unique values.